gatsby/src/components/content-box/content-box.tsx

```diff
@@ -41,7 +43,13 @@ const ContentBox: React.FC<Props> = ({
   }, [title]);
 
   return (
-    <div className={classNames(styles.contentBox, 'row')}>
+    <div
+      className={classNames(
+        styles.contentBox,
+        styles[`contentBox--${variant}`],
```

Comment:
please, make sure PurgeCSS does not remove it. https://cesko-digital.slack.com/archives/C01DVGTNRUL/p1604246550476900